### PR TITLE
assist #9610: adjust rules for moving link children

### PR DESCRIPTION
**Pull request by mtbc**
Marks three integration tests as no longer broken. Integration tests should still pass. Could look for regressions regarding moving images that have owner's and others' comments and tags between groups.

### CODE REVIEW

File: components/tools/OmeroJava/test/integration/chgrp/AnnotationMoveTest.java [hunk 1]
```diff
@@ -96,7 +96,9 @@ private void moveImageWithNonSharedAnnotation(String src, String dest,
         assertEquals(iQuery.findAllByQuery(sb.toString(), param).size(),
                 annotationIdsUser1.size());
         n = 0;
-        if (src.equals("rwrw--")) n = annotationIdsUser2.size();
+        if (src.equals("rwrw--") && !dest.equals("rw----")) {
```
Comment (mtbc) [inline components/tools/OmeroJava/test/integration/chgrp/AnnotationMoveTest.java]: We do not expect to see another user's annotations once we have moved the image into a private group.

**Comment by pwalczysko** [comment]
Tried to move entities tagged with owners tags (in ro) and other's tags (in ra) to another group. Works as expected.

**Comment by pwalczysko** [comment]
Waiting for integration tests builds to finish.

**Comment by mtbc** [comment]
I don't *think* anything in the current crop of failures points to chgrp.

**Comment by pwalczysko** [comment]
*-integration-java is green https://ci.openmicroscopy.org/view/5.1/job/OMERO-5.1-merge-integration-java/

**Comment by pwalczysko** [comment]
*-python tests are yellow with four failed tests, I have to rely on @mtbc when he says they have nothing to do with this PR. https://ci.openmicroscopy.org/view/5.1/job/OMERO-5.1-merge-integration-python/340/ 

All seems fine here.

**Pull request merged**